flappy-bird.js

```diff
@@ -24,16 +25,42 @@ class Cube extends Shape {
     }
 }
 
+export class Test_Data {
```

Comment:
Remove unnecessary parts of this class. We don't need all the attributes provided here and please come up with a better name to represent the class to make the code more readable